doc/user/zebra.rst

```diff
@@ -351,6 +360,143 @@ default) should the specified gateways not be reachable. Eg:
 
 .. _multicast-rib-commands:
 
+VRF Commands
+============
+
+Currently, the user has the possibility to configure VRFs. VRF is a way to
+separate networking contexts on the same machine. Those networking contexts
+stick to separate interfaces, thus making it possible to associate one interface
+to a specific VRF.
+VRF can be used, for example, when instantiating per enterprise networking
+services, without having to instantiate the physical host machine or the
+routing management daemons for each enterprise. As a result, interfaces are
+separate for each set of VRF, and routing daemons can have their own context
+for each VRF.
+
+This conceptual view introduces the `Default VRF` case. If the user does not
+configure any specific VRF, then by default, the user will however configure
+the `Default VRF`.
+On the Zebra context, this can be done when being in configuration mode, when
+configuring a static route `ip route NETWORK GATEWAY`.
+
+   .. code-block:: frr
+
+      # case without VRF
+      configure terminal
+      ip route 10.0.0.0 255.255.255.0 10.0.0.2
+      exit
+
+Configuring VRF networking contexts can be done on various ways on FRR.
+The VRF interfaces can be configured by entering in interface configuration
+mode : `interface INTERFACE VRF`. Also, if the user wants to configure a static
+route for a specific VRF, then a specific VRF configuration mode is available.
+After entering into that mode by following command: `vrf VRF`. the user can
+enter the same route command as before, but this time, the route command
+will apply to vrf VRF.
+
+   .. code-block:: frr
+
+      # case with VRF
+      configure terminal
+      vrf r1-cust1
+       ip route 10.0.0.0 255.255.255.0 10.0.0.2
+      exit-vrf
+
+A VRF backend mode is chosen when running zebra.
+
+If no option is chosen, then the `Linux VRF lite` implementation
+will be mapped over the Zebra VRF. The routing table associated to
+that VRF is a Linux table identifier located in the same `Linux
+network namespace` where Zebra started.
+
+If the -n option is chosen, then the `Linux network namespace`
+will be mapped over the Zebra VRF. That implies that Zebra is
+able to configure several `Linux network namespaces`.  The routing
+table associated to that VRF is the whole routing tables located
+in that namespace. For instance, this mode matches Opentack or 6WIND
+Linux vRouters ; others have reported to use this mode too. The
+default behavior remains Linux VRF lite that os widely used by Cumulus
```

Comment:
I'm not sure we need to spell out who does what here.  Linux VRF lite is seen as the way to do vrf's by the linux kernel community.